dotcom-rendering/package.json

```diff
@@ -184,6 +184,7 @@
     "stylelint-config-styled-components": "^0.1.1",
     "stylelint-processor-styled-components": "^1.10.0",
     "swr": "^1.3.0",
+    "to-string-loader": "^1.2.0",
```

Comment:
We referenced 'to-string-loader' in our webpack browser bundle, but it wasn't actually installed! This lets us get the contents of the `debug.css` file and add it to the document!